mkosi.md

```diff
@@ -1322,6 +1328,26 @@ turn-around times for complete image builds are minimal, as only
 changed source files need to be recompiled: an OS image rebuilt will
 be almost as quick to build the source tree only.
 
+# PRESETS
+
+Presets can be defined in mkosi.presets/. A preset is just like a
```

Comment:
Backticks around the path, here and below.

I think this text could be a bit more clear.
1. what is a preset?
2. where are presets located and how are they read?
3. how are presets ordered?
4. how are presets executed?
5. comments and examples